package-lock.json

```diff
@@ -5,142 +5,147 @@
-        "chalk": "2.2.2",
-        "circular-dependency-plugin": "5.0.2",
-        "clean-css": "4.1.11",
-        "copy-webpack-plugin": "4.5.1",
-        "file-loader": "1.1.11",
-        "glob": "7.1.2",
-        "html-webpack-plugin": "3.2.0",
-        "istanbul": "0.4.5",
-        "istanbul-instrumenter-loader": "3.0.1",
-        "karma-source-map-support": "1.3.0",
-        "less": "3.0.4",
-        "less-loader": "4.1.0",
-        "license-webpack-plugin": "1.3.1",
-        "lodash": "4.17.10",
-        "memory-fs": "0.4.1",
-        "mini-css-extract-plugin": "0.4.0",
-        "minimatch": "3.0.4",
-        "node-sass": "4.9.0",
-        "opn": "5.3.0",
-        "parse5": "4.0.0",
-        "portfinder": "1.0.13",
-        "postcss": "6.0.22",
-        "postcss-import": "11.1.0",
-        "postcss-loader": "2.1.5",
-        "postcss-url": "7.3.2",
-        "raw-loader": "0.5.1",
-        "resolve": "1.7.1",
-        "rxjs": "6.2.2",
-        "sass-loader": "7.0.3",
-        "silent-error": "1.1.0",
-        "source-map-support": "0.5.6",
-        "stats-webpack-plugin": "0.6.2",
-        "style-loader": "0.21.0",
-        "stylus": "0.54.5",
-        "stylus-loader": "3.0.2",
-        "tree-kill": "1.2.0",
-        "uglifyjs-webpack-plugin": "1.2.5",
-        "url-loader": "1.0.1",
-        "webpack": "4.8.3",
-        "webpack-dev-middleware": "3.1.3",
-        "webpack-dev-server": "3.1.4",
-        "webpack-merge": "4.1.2",
-        "webpack-sources": "1.1.0",
-        "webpack-subresource-integrity": "1.1.0-rc.4"
+      "version": "0.7.1",
+      "resolved": "https://registry.npmjs.org/@angular-devkit/build-angular/-/build-angular-0.7.1.tgz",
+      "integrity": "sha512-GolB7Av3zTfQ23wDbuMR1b049vRlEVGtfIQCATE05yGmet+w+sogN2/YgRX3dPtG/tEhjJTbTIe5kV0RCVVlqQ==",
+      "dev": true,
+      "requires": {
+        "@angular-devkit/architect": "0.7.1",
+        "@angular-devkit/build-optimizer": "0.7.1",
+        "@angular-devkit/build-webpack": "0.7.1",
+        "@angular-devkit/core": "0.7.1",
+        "@ngtools/webpack": "6.1.1",
+        "ajv": "~6.4.0",
+        "autoprefixer": "^8.4.1",
+        "circular-dependency-plugin": "^5.0.2",
+        "clean-css": "^4.1.11",
+        "copy-webpack-plugin": "^4.5.2",
+        "file-loader": "^1.1.11",
+        "glob": "^7.0.3",
+        "html-webpack-plugin": "^3.0.6",
+        "istanbul": "^0.4.5",
+        "istanbul-instrumenter-loader": "^3.0.1",
+        "karma-source-map-support": "^1.2.0",
+        "less": "^3.0.4",
+        "less-loader": "^4.1.0",
+        "license-webpack-plugin": "^1.3.1",
+        "mini-css-extract-plugin": "~0.4.0",
+        "minimatch": "^3.0.4",
+        "node-sass": "^4.9.1",
+        "opn": "^5.1.0",
+        "parse5": "^4.0.0",
+        "portfinder": "^1.0.13",
+        "postcss": "^6.0.22",
+        "postcss-import": "^11.1.0",
+        "postcss-loader": "^2.1.5",
+        "postcss-url": "^7.3.2",
+        "raw-loader": "^0.5.1",
+        "rxjs": "^6.0.0",
```

Comment:
why are we getting the caret symbol '^'?  rxjs": "^6.0.0" ?

Reply:
That is what is defined by `@angular-devkit/build-angular` in it's `package.json` file. It has nothing to do with our locked version. For each of these dependencies, you have an entry in this file, with a fixed version.